lib/v2/api.js

```diff
@@ -88,7 +88,7 @@
          * @param {Function} [cb(data)] Callback
          */
         apply: function(element, cb) {
-            var teamslug = element.getAttribute('data-gratipay-teamslug');
+            var projectslug = element.getAttribute('data-gratipay-teamslug');
```

Comment:
I'm putting in the option for `var projectid = element.getAttribute('data-gratipay-project-id');` though.